ql/src/test/org/apache/hadoop/hive/ql/io/parquet/serde/TestParquetTimestampUtils.java

```diff
@@ -389,6 +390,26 @@ public void testIllegalInt64TimestampStrings() {
     verifyInt64TimestampValue("2262-04-11 23:47:16.854775808", LogicalTypeAnnotation.TimeUnit.NANOS, false);
   }
 
+  @Test
```

Comment:
The test passes successfully if you exclude all the changes in this PR and revert some lines in `NanoTimeUtils` as I mentioned in the JIRA case. This makes me think that we need to treat the problem differently since it does not appear to be related to the formatter.

Reply:
What I wrote above is not true. Reverting the changes about the proleptic calendar in `NanoTimeUtils` does not fix this test. I don't know why the test was passing before but I suspect a "stale" workspace.